spec/jobs/rapid_ready_for_decision/form526_asthma_job_spec.rb

```diff
@@ -51,6 +51,10 @@
       it 'returns the active medication requests' do
         expect(subject[:medications].count).to eq(19)
       end
+
+      it 'flags potential asthma-related medication' do
+        expect(subject[:medications].select { |med| med[:flagged] }.count).to eq(3)
```

Comment:
This is for an above line, but saw that Yoom started adding the `asthma_claim_for_increase` trait when doing `create(:form526_submission, user: user)`

Reply:
Would love to figure out the best way to use this those test features (this spec does not rely on typical Rails models)